packages/manager/src/hooks/useAdobeAnalytics.ts

```diff
@@ -4,6 +4,7 @@ import React from 'react';
 
 import { ADOBE_ANALYTICS_URL } from 'src/constants';
 import { reportException } from 'src/exceptionReporting';
+import { getStoredCustomerUuid } from 'src/utilities/analytics/utils';
```

Comment:
```suggestion
import { getStoredCustomerUUID } from 'src/utilities/analytics/utils';
```

I think we should use `getStoredCustomerUUID (or getStoredCustomerEuuid, whichever is appropriate) and update all references in this PR for consistency